doc/advanced-topics.md

```diff
@@ -102,6 +103,27 @@ accepted. Provided the `type` is provisioned correctly, the `value` may be defin
 }
 ```
 
+Because GeoJSON types (e.g. `Point`, `LineString` etc.) are native types in **NGSI-LD**, automatic GeoJSON conversion is switched on for NGSI-LD by default.
+
+With **NGSI-v2**, for backwards compatibility reasons, automatic GeoJSON conversion for types other than `geo:json` is turned off by default.
+Add the `autocastGeoJSON` configuration to the attribute to enable GeoJSON conversion. Each GeoJSON attribute can be provisioned as shown:
+
+```json
```

Comment:
This example in documentation is for a configuration group. However, the templates modified in this PR are only about devices:

The behavior should be as follow:

![imagen](https://user-images.githubusercontent.com/1534240/116227265-3d272880-a754-11eb-8fb3-e0a857601b0d.png)

* It can be enabled/disabled at configuration group level
* It can be enabled/disabled at device level. If not defined at device level, the config group setting is used (and if not defined at config group level, then assume false by default)